.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+<!-- Provide a description of what has been changed -->
+
+### Checklist
+
+- [ ] A PR is opened to update the documentation on https://github.com/kedacore/keda-docs
+- [ ] Tests have been added
+
+
+Relates to #
```

Comment:
Might want to use any of these keywords (eg. Fixes), https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords#about-issue-references

Reply:
I used relates to so that the issue is linked but not closed for every PR, but happy to change.

`fixes` then?

Reply:
I don't have a strong opinion, it's up to you :) You can keep `relates`, put `fixes` or you can add both options to the template and user can choose. Whatever works for me, not a big deal.